crypto/common/fdoKeyExchange.c

```diff
@@ -20,8 +20,8 @@ static int32_t remove_java_compatible_byte_array(fdo_byte_array_t *BArray);
 /**
  * fdo_kex_init() - Initialize key exchange context
  * Initialize the key exchange context which can be done at init time
- * o Key Exchange algorithm (DH, AYSM, ECDH, ECDH384)
- * o Cipher Suite to be used (AESxxx/CTRorCBC/HMAC-SHAyyy)
+ * o Key Exchange algorithm (ECDH, ECDH384)
+ * o Cipher Suite to be used
```

Comment:
why is the CS options removed? Cipher Suite set to value indicated by compiler flags?

Reply:
Cipher suite is set at compilation time, but the comment can be fixed in a subsequent PR.